tools/packaging/common/envoy_bootstrap_v2.json

```diff
@@ -49,11 +49,94 @@
       {
         "tag_name": "mongo_prefix",
         "regex": "^mongo\\.(.+?)\\.(collection|cmd|cx_|op_|delays_|decoding_)(.*?)$"
+      },
+      {
+          "regex": "(reporter=\\.=(.+?);\\.;)",
```

Comment:
these regexs are pretty complex. Should we have a test for them? The existing regex ones are worse

Reply:
For sure we will need an integration test, but I'd rather not to do it in this PR. I create a issue https://github.com/istio/istio/issues/16722 to track this and will work on it after this PR merged.